crates/db-micro-http/README.md

```diff
@@ -0,0 +1,63 @@
+# db-micro-http
+
+## Design
+
+db-micro-http is a minimal implementations of HTTP 1.0 and HTTP 1.1 protocols. 
```

Comment:
It should explicitly state the relationship and acknowledge AWS firecracker team.
And state why it's forked here.

Reply:
Added. Thanks :)